src/app/home-page/components/home-page/home-page.component.html

```diff
@@ -1,21 +1,28 @@
-<app-slideshow></app-slideshow>
-<app-heading [alertText]="'new'" [headingText]="'Arrivals'"></app-heading> 
+<app-slideshow [slideShowImages]="slideShowImages"></app-slideshow>
+<app-heading [alertText]="'new'" [headingText]="'Arrivals'"></app-heading>
 <app-carousel *ngIf="products.length">
   <app-carousel-item *ngFor="let product of products">
     <app-product-item (addItemToWishList)="wishListHandler($event)" [product]="product"></app-product-item>
   </app-carousel-item>
 </app-carousel>
 <app-swipe-icon></app-swipe-icon>
 <div class="main-container">
-  <div class="row-sb">
-    <app-advertising-area *ngFor="let banner of banners" [content]="banner.htmlSnippet"></app-advertising-area>
+  <div class="row-sb" *ngIf="banners.length">
```

Comment:
apply the same behavior as the slideshow, because banners have placeholders as well